Fix slow tests

slow tests can be run on Github Actions with https://github.com/snek5000/snek5000-cbox/actions/workflows/tests_slow.yml

`test_SW_linear_base_from_SFD` is now broken and needs to be fixed.

```
    @pytest.mark.slow
    def test_SW_linear_base_from_SFD():
    
        params = params_SW()
    
        params.Ra_side = 1.86e8
    
        params.oper.enable_sfd = float(True)
    
        params.nek.general.end_time = 900
        params.nek.general.write_interval = 10
    
        aspect_ratio = 1.0
        nb_elements = 10
        params.oper.ny = nb_elements
        params.oper.nx = int(nb_elements / aspect_ratio)
    
        sim_sfd = Simul(params)
    
>       sim.make.exec("run_fg", nproc=4)
E       UnboundLocalError: local variable 'sim' referenced before assignment
```

Other error: (@akhoubani )
```
     @pytest.mark.slow
    def test_SW_linear_base_provided():
    
        params = params_SW()
    
        restart_file = "./doc/examples/base_flow_side_simple.restart"
    
        params = params_SW()
    
        params.Ra_side = 1.86e8
    
        aspect_ratio = 1.0
        nb_elements = 10
        params.oper.ny = nb_elements
        params.oper.nx = int(nb_elements / aspect_ratio)
    
        params.nek.general.start_from = "base_flow.restart"
    
        params.nek.problemtype.equation = "incompLinNS"
        params.oper.elem.staggered = "auto"
    
        params.nek.general.extrapolation = "standard"
    
        sim = Simul(params)
    
        copyfile(restart_file, sim.params.output.path_session / "base_flow.restart")
    
        sim.make.exec("run_fg", nproc=4)
    
        growth_rate_linear = compute_growth_rate(sim)
    
>       assert 0.0049 < growth_rate_linear < 0.0055
E       assert 0.008127315550479523 < 0.0055

```

--- a/src/snek5000_cbox/cbox.usr.f
+++ b/src/snek5000_cbox/cbox.usr.f
@@ -80,7 +80,9 @@
       enable_sfd = UPARAM(7)
 
       if (ISTEP.eq.0) then
-         ! TIME = 0
+         if (IFPERT) then
+            TIME = 0
+         endif   
       ! start framework
          call frame_start
       endif

